### DISCUSSION TITLE
Camera package for Ebitengine

### DISCUSSION THREAD

**Discussion by setanarut** [sedyh/awesome-ebiten#45] [Show and tell]
I've released v2 of my camera package, if you want to add it to the [World](https://github.com/sedyh/awesome-ebitengine?tab=readme-ov-file#world) list you can take a look. https://github.com/setanarut/kamera

### Run example on your local machine

```console
go run github.com/setanarut/kamera/v2/examples/demo@latest
```

**Comment by sedyh** [comment]
Hello, thanks for PR. Looks great, added.
